src/main/java/me/jellysquid/mods/sodium/client/render/chunk/compile/ChunkBuilder.java

```diff
@@ -192,7 +244,11 @@ public void init(ClientWorld world, BlockRenderPassManager renderPassManager) {
             throw new NullPointerException("World is null");
         }
 
-        this.stopWorkers();
+        if (this.running.get()) {
```

Comment:
I added this mostly for debugging purposes. However, unless another mod is directly calling in to `init`, I'm pretty sure this predicate is accurate. There is only one reference to `init` that I can find in IntelliJ and it's directly after the constructor call.